src/components/modules/toolbar/blockSettings.ts

```diff
@@ -158,7 +176,10 @@ export default class BlockSettings extends Module {
       return this.buttons;
     }
 
-    const toolSettings = this.nodes.toolSettings.querySelectorAll(`.${this.Editor.StylesAPI.classes.settingsButton}`);
+    const toolSettings = this.nodes.toolSettings.querySelectorAll(
+      // Select buttons and inputs
+      `.${StylesAPI.classes.settingsButton}, .${StylesAPI.classes.input}`
```

Comment:
why not use `$.findAllInputs()` here?

Reply:
can we write a method for accepting `cdx-buttons` and every-inputs with saving their order at dom?